draft-ietf-ccwg-rfc5033bis

```diff
@@ -103,23 +105,26 @@ congestion control algorithms and for the IETF community when evaluating whether
 a proposal is appropriate for publication in the RFC series and for
 deployment in the Internet.
 
-This document obsoletes the similarly titled {{?RFC5033}} that was
+This document obsoletes the similarly titled {{?RFC53}} that was
 published in 2007 as a Best Current Practice to evaluate alternate
 congestion control algorithms as Experimental or Proposed Standard RFCs.
 
 The IETF's standard congestion control algorithms have been shown to have
 performance challenges in various environments
 (e.g., high-speed networks, cellular and WiFi wireless technologies,
 long distance satellite links)
-and also for specific traffic workloads (VoIP, gaming, and videoconferencing).
+and also flows carrrying specific workloads (VoIP, gaming, and videoconferencing).
 
-In 2007, TCP was the dominant consumer of this work, and proposals were
-typically discussed in research groups, for example the
-Internet Congestion Control Research Group (ICCRG).
+In 2007, TCP was the dominant consumer of this work, and proposals
+were typically discussed in the Internet Congestion Control Research Group (ICCRG).
+The Datagream Congestrion Copntrol Protocol (DCCP)
+was developed as a method for developing congestion control algorithms for
+datagram flows.
 
-Since RFC 5033 was published, many conditions have changed.
+Since RFC 53 was published, many conditions have changed.
 The set of protocols using these algorithms has spread beyond
-TCP and SCTP to include DCCP, QUIC, and beyond.
+TCP, SCTP {{?RFC9260}}, and DCCP {{?RFC4340}}, to include QUIC {{?RFC9000}},
+RTP Media Congestion Avoidance Techniques (RMCAT) and beyond.
 Some proponents of alternative congestion control algorithms now have the opportunity
```

Comment:
Pleas read new text and see if this improved.

Reply:
It is better. The only part that I don't like is the Oxford comma and the double "and" list. You can get the meaning when you read it, but not when you hear it. I would prefer something like:

"Since RFC 5033 was published, many conditions have changed. In addition to 
The set of protocols using these algorithms has spread beyond
TCP, SCTP and DCCP. It now includes QUIC {{?RFC9000}},
RTP Media Congestion Avoidance Techniques (RMCAT) and probably more."

Reply:
Text updated